package.c

```diff
@@ -0,0 +1,109 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <ctype.h>
+#include <curl/curl.h>
+
+#include "ucentral.h"
+
+int escapePackageName(const char *name) {
```

Comment:
can you kindly convert the code to not use camelCase style naming ?
as in
escapePackageName -> escape_package_name
validatePackageURL -> validate_package_URL
....